apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationPreviewList.tsx

```diff
@@ -0,0 +1,578 @@
+      }
+      areaOfSelection {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      processNumber
+      poolSkills {
+        id
+        type {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        skill {
+          id
+          key
+          name {
+            en
+            fr
+          }
+          category {
+            value
+            label {
+              en
+              fr
+            }
+          }
+        }
+      }
+      screeningQuestions {
+        id
+      }
+    }
+  }
+`);
+
+interface ReviewApplicationDialogProps {
+  applicationQuery: ReviewApplicationPreviewListFragment;
+}
+
+const ReviewApplicationDialog = ({
+  applicationQuery,
+}: ReviewApplicationDialogProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const application = applicationQuery;
+  const pool = application?.pool;
+
+  const nullMessage = intl.formatMessage(commonMessages.notFound);
+  const poolName = getLocalizedName(pool?.name, intl);
+
+  // Separate essential and asset skills, sort them by category, and confirm they include skill data
+  const poolSkills = unpackMaybes(pool?.poolSkills);
+  const essentialPoolSkills = sortPoolSkillsBySkillCategory(
+    poolSkills.filter(
+      (poolSkill) => poolSkill.type?.value === PoolSkillType.Essential,
+    ),
+  );
+  const nonessentialPoolSkills = sortPoolSkillsBySkillCategory(
+    poolSkills.filter(
+      (poolSkill) => poolSkill.type?.value === PoolSkillType.Nonessential,
+    ),
+  );
+
+  const focusOnRecruitment = useRef(false);
+
+  const status = getApplicationStatusChip(
+    application.submittedAt,
+    pool.closingDate,
+    application.removedAt,
+    application.finalDecisionAt,
+    application.finalDecision?.value,
+    pool.areaOfSelection?.value,
+    application.assessmentStatus,
+    pool.screeningQuestions,
+    intl,
+  );
+
+  const isDraftStatus = status.value === ApplicationStatus.DRAFT;
+  const isExpiredStatus = status.value === ApplicationStatus.EXPIRED;
+  const lessThanThreeDaysTillClosingDate = pool?.closingDate
+    ? differenceInDays(Date.now(), parseDateTimeUtc(pool.closingDate)) < 3
+    : null;
+  const showDeadlineToApply =
+    isDraftStatus ||
+    (isDraftStatus && lessThanThreeDaysTillClosingDate) ||
+    isExpiredStatus;
+
+  const isSuccessfulStatus = status.value === ApplicationStatus.SUCCESSFUL;
+
+  return (
+    <Dialog.Root open={isOpen} onOpenChange={setIsOpen}>
+      <Dialog.Trigger asChild>
+        <PreviewList.Button label={poolName} />
+      </Dialog.Trigger>
+      <Dialog.Content
+        onCloseAutoFocus={(e) => {
+          if (focusOnRecruitment.current) {
+            e.preventDefault();
+            document.getElementById(`${application.id}-test`)?.focus();
+          }
+
+          focusOnRecruitment.current = false;
+        }}
```

Comment:
Hmm, this makes me wonder if there is a bug somewhere else. Do you remember why this was necessary?

Reply:
I think it had to do with the structure of the PreviewList component, but I did do this a month ago so it's hard to remember the specifics :sweat_smile:

Reply:
Oh ya it's for this requirement:
- For Successful, there is a "View recruitment process" button/link that **Closes the dialog**, **Expands the processes tool**, and **focuses the related process**

There's a ticket that would build on top of this to open the accordion then focus on the related process. However, I remember we talked (awhile ago now :laughing:) that this would most likely present an be accessibility issue. 

If we remove this then we would have to remove the button from the dialog as well (it appears on **successful** applications). I'll need to talk to the designers about that.

Reply:
I'm not sure that you described is possible which is the issue. Regardless of all that though, we need to change the cards to differentiate between Applications and Recruitment processes. 

Right now, the buttons are identical even though they open different dialogs.